applications/submit_data_from_external.py

```diff
@@ -55,34 +53,37 @@ def _parse(label):
 
     """
 
-    config = configurator.Configurator(label=label)
+    config = Configurator(label=label, description=description, usage=usage)
 
     config.parser.add_argument(
-        "-m",
-        "--input_meta_file",
-        help="User-provided meta data file (yml)",
+        "--input_meta",
+        help="Meta data file describing input data",
         type=str,
-        required=True,
+        required=False,
     )
     config.parser.add_argument(
-        "-d",
-        "--input_data_file",
-        help="User-provided data file (ecsv)",
+        "--input_data",
+        help="Input data file",
         type=str,
-        required=True,
+        required=False,
     )
     return config.initialize(workflow_config=True)
 
 
 def main():
 
     label = os.path.basename(__file__).split(".")[0]
```

Comment:
This is an issue with the new approach for configuration parameters: we can't use the required flags, as we don't know at which step this parameter is set (e.g, if not on the command line, it might be provided in a config file). 

This is something to be improved in future.